nima/webserver/webserver/src/main/java/io/helidon/nima/webserver/LoomServer.java

```diff
@@ -238,15 +270,19 @@ private void stopIt() {
 
     private void startIt() {
         long now = System.currentTimeMillis();
-        boolean result = parallel("start", ServerListener::start);
-        if (!result) {
-            LOGGER.log(System.Logger.Level.ERROR, "Níma server failed to start, shutting down");
-            parallel("stop", ServerListener::stop);
-            if (startFutures != null) {
-                startFutures.forEach(future -> future.future().cancel(true));
-            }
-            return;
+        // TODO parallel start breaks pico - issue
+        // boolean result = parallel("start", ServerListener::start);
+        for (ServerListener listener : listeners.values()) {
+            listener.start();
         }
+        //        if (!result) {
```

Comment:
remove

Reply:
There is a follow up for a problem where Pico does not work when server is started in parallel. This is a temporary workaround of that problem, not the final solution